### PR TITLE
Cirq for Quantum Computing

### LINKED ISSUE #7378: Cirq for Quantum Computing
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Cirq for Quantum Computing

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Cirq is an open-source framework built by the Google AI Quantum Team in 2018. It is a Python software library for writing, manipulating, and optimizing quantum circuits, and then running them on quantum computers and quantum simulators. Cirq provides useful abstractions for dealing with today’s noisy intermediate-scale quantum computers, where details of the hardware are vital to achieving state-of-the-art results.
This tutorial will give an overview of the framework and demonstrate to our readers how they can use this framework to learn about Quantum computing.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

- An Overview of Cirq.
- Learn about essential concepts of the framework.
- Implement an example to demonstrate how to use the framework.

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

Quantum computing is still in its early stages. It might be hard to grasp concepts about Quantum computing. With open-source frameworks such as Cirq, learning about quantum computing is easier. This tutorial will simplify the learning process by demonstrating to our readers how they can get started learning about Quantum computing.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Cirq](https://quantumai.google/cirq).
- [GitHub](https://github.com/quantumlib/cirq).

### PR DESCRIPTION
**Pull request by splashtonia**
This closes #7378

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/cirq-for-quantum-computing/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+simulator = cirq.Simulator()
+output = simulator.run(q_circuit, repetitions = 20)
+print(output)
+```
+
+```bash
+x,y=11011000000111111111, 11011000000111111111
+```
+
+Lets add an `X` gate to qubit `y` and see whether our result will be different.
```
Comment (wilkistermumbi) [inline content/articles/cirq-for-quantum-computing/index.md:102]:
```suggestion
Let's add an `X` gate to qubit `y` and see whether our result will be different.
```

File: content/articles/cirq-for-quantum-computing/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+There are several Google Quantum Hardware that has been used by the Google Research team to date. [Foxtail](https://quantumai.google/cirq/google/devices#foxtail) is one of the hardware devices that was released in 2016, [Bristlecone](https://quantumai.google/cirq/google/devices#bristlecone) is a 72-qubit device and was released in 2018, and most recently, [Sycamore](https://quantumai.google/cirq/google/devices#sycamore) released in 2019. Sycamore is the 54-qubit quantum hardware device that was used to reach the quantum advantage mentioned above. These results are published in the, [Quantum supremacy using a programmable superconducting processor](https://www.nature.com/articles/s41586-019-1666-5) research paper in the Nature journal.  
+
+Cirq is a quantum computing programming language. It is an open-source, Python framework for writing, manipulating, and optimizing quantum circuits. It can be used to run these programs on simulators and real quantum computers.
```
Comment (wilkistermumbi) [inline content/articles/cirq-for-quantum-computing/index.md:36]:
```suggestion
Cirq is a quantum computing programming language. It is an open-source Python framework for writing, manipulating, and optimizing quantum circuits. It is used to run these programs on simulators and real quantum computers. 
```

File: content/articles/cirq-for-quantum-computing/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+There are several Google Quantum Hardware that has been used by the Google Research team to date. [Foxtail](https://quantumai.google/cirq/google/devices#foxtail) is one of the hardware devices that was released in 2016, [Bristlecone](https://quantumai.google/cirq/google/devices#bristlecone) is a 72-qubit device and was released in 2018, and most recently, [Sycamore](https://quantumai.google/cirq/google/devices#sycamore) released in 2019. Sycamore is the 54-qubit quantum hardware device that was used to reach the quantum advantage mentioned above. These results are published in the, [Quantum supremacy using a programmable superconducting processor](https://www.nature.com/articles/s41586-019-1666-5) research paper in the Nature journal.  
+
+Cirq is a quantum computing programming language. It is an open-source, Python framework for writing, manipulating, and optimizing quantum circuits. It can be used to run these programs on simulators and real quantum computers.   
+
+### Implementing an example
+To get Cirq installed in your notebook, we'll perform a quick pip install.
```
Comment (wilkistermumbi) [inline content/articles/cirq-for-quantum-computing/index.md:39]:
```suggestion
To get Cirq installed in your notebook, we will perform a quick `pip` install.
```

**Comment by wilkistermumbi** [comment]
@splashtonia Please work on the edits before we can move on for final review.

**Comment by splashtonia** [comment]
@wilkistermumbi done.

**Comment by hectorkambow** [comment]
jumping on this one @wilkistermumbi 

@splashtonia

**Comment by hectorkambow** [comment]
👏  

nicely done @splashtonia  
Thank you for your patience while we clear out the backlog via the old process - moving forward: future articles will be approved/reviewed much quicker w/ a newer process/ 
Be sure to see you published work live on the site.

PS: I'd also recommend to add to your skills sections (this will go a long way with potential employers).

**Pull request merged**